src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = this.hidden;
```

Comment:
```suggestion
    this.hidden = !this.hidden;
```

Reply:
In my work is look like this -> this.hidden = !this.hidden;
but when I push it, it becomes without '!'

Reply:
Now it's corrected )